wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,57 +1,83 @@
-  - Getting a 100 for a slider requires getting a 100 judgement or better for the slider head.
+| Judgement | Slider completion |
+| :-: | :-- |
+| Great | 100% |
+| Ok | 50% |
+| Meh | At least one slider part |
+| Miss | 0% |
+
+The slider head only needs to be hit within the Meh hit window for a successful hit. However, if [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) is active, slider head accuracy is additionally accounted for, with the judgement received for the entire slider being limited to at most the one received for the slider head.
 
-The slider parts also have additional influence on the current [combo](/wiki/Gameplay/Combo_(score_multiplier)):
+There are some additional quirks with how sliders influence [combo](/wiki/Gameplay/Combo_(score_multiplier)):
 
-- Tapping the slider head too early does not incur a miss, but will reset combo to 0.
-- Missing a slider tick or repeat does not incur a miss, but will reset combo to 0.
-- Missing the slider end does not incur a miss, but will not increment combo.
+- Tapping the slider head too early (before the Meh hit window), missing a slider tick, or missing a repeat does not incur a Miss, but will cause a [combo break](/wiki/Gameplay/Judgement/Combobreak). The other slider parts can still be hit if a key is held down. This is colloquially referred to as a [slider break](/wiki/Gameplay/Judgement/Slider_break).
+- Missing the slider end does not incur a Miss, but will not increment combo.
 
 ### Spinners
 
-Each spinner has a set number of spins required to complete it. This number depends on the [overall difficulty](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners) of the beatmap. The criteria for judging the spinner as a whole are as follows:
+Each spinner has a set number of spins required to complete it. This number depends on the [overall difficulty](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners) of the beatmap.
+
+| Judgement | Spins required |
+| :-: | :-- |
+| Great | 100% |
+| Ok | One spin less than the required number |
+| Meh | 25% |
+| Miss | 0% |
+
+The required spins depends on the duration of the spinner and the overall difficulty. The spins per second required are given as follows:
+
+<!-- TODO: needs verification -->
 
-- 300: The player successfully performed the number of spins required for completion, or more.
-- 100: The player performed one spin less than the required number.
-- 50: The player performed at least 25% of the required spins.
-- Miss: The player failed to meet the minimum of 25% of spins required.
+| OD | Spins per second |
+| :-: | :-- |
+| < 5 | `5 - 2 * (5 - OD) / 5` |
+| = 5 | `5` |
+| > 5 | `5 + 2.5 * (OD - 5) / 5` |
 
-For some very short spinners, the number of spins required is actually calculated to be 0, and thus the spinner always automatically completes itself with a 300.
+If a spinner is very short, the number of spins required may be calculated to be 0, and thus the spinner will always complete itself with a Great.
```

Comment:
this got me confused for a few second because I always remember [wizards in winter](https://osu.ppy.sh/beatmapsets/132#osu/296), but then I read the historical note below. mind adding something like "this has been added in ..."?

Reply:
i don't want to bloat the article with a lot of historical info in between descriptions of actual behaviour, it's fine as is

almost thought the same for the geki/katu explanation but it happened to be kinda relevant and informative, but i did move it a bit out of the way as i hope you can tell